hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -118,7 +118,8 @@ define([
                 var deffered = $q.defer();
 
                 API.resource('HRPayScale','get',{
-                    sequential: 1
+                    sequential: 1,
+                    is_active: 1
```

Comment:
Job contracts doesn't have much test coverage, so I think it would be great if you could add a test to cover this change.